README.md

```diff
@@ -122,6 +124,8 @@ Help / Documentation
 
 - API reference documentation: [![Read the documentation of the stable version](https://readthedocs.org/projects/pip/badge/?version=stable)](http://bqplot.readthedocs.org/en/stable/) [![Read the documentation of the development version](https://readthedocs.org/projects/pip/badge/?version=latest)](http://bqplot.readthedocs.org/en/latest/)
 
+- Talk to us on the Gitter chat: [![Join the chat at https://gitter.im/bloomberg/bqplot](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/bloomberg/bqplot?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Let's add both, and say you can ask for widget help at the ipywidgets chat?

If the bqplot gitter becomes fully defunct, we can formally love it. Feel free to merge

----- Original Message -----
From: Sylvain Corlay <reply@reply.github.com>
To: bqplot@noreply.github.com
CC: DHRUV MADEKA, author@noreply.github.com
At: 23-Jun-2016 05:32:01


In README.md: > @@ -122,6 +124,8 @@ Help / Documentation
>  
>  - API reference documentation: [![Read the documentation of the stable version](https://readthedocs.org/projects/pip/badge/?version=stable)](http://bqplot.readthedocs.org/en/stable/) [![Read the documentation of the development version](https://readthedocs.org/projects/pip/badge/?version=latest)](http://bqplot.readthedocs.org/en/latest/)
>  
> +- Talk to us on the Gitter chat: [![Join the chat at https://gitter.im/bloomberg/bqplot](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/bloomberg/bqplot?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
Just that I am already in more than 30 gitter rooms and I have less chances to miss things if widgets related things are together (and posts on bqplot are rare anyway...) Up to you.  
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.